$NetBSD: patch-rebar.config,v 1.3 2021/05/21 06:28:37 triaxx Exp $

On NetBSD and OpenBSD there is no libdl, as the required functionality is
already built into libc.

--- rebar.config.orig	2021-03-24 17:32:32.000000000 +0000
+++ rebar.config
@@ -26,7 +26,7 @@
 
 {port_env, [{"ERL_LDFLAGS", " -L$ERL_EI_LIBDIR -lei"},
             {"CFLAGS", "$CFLAGS"},
-            {"^((?!openbsd).)*$", "LDFLAGS", "$LDFLAGS -ldl"},
+            {"^((?!(net|open)bsd).)*$", "LDFLAGS", "$LDFLAGS -ldl"},
             {"LDFLAGS", "$LDFLAGS"}]}.
 
 {port_specs, [{"priv/bin/eimp", ["c_src/eimp.c"]}]}.
